### PR TITLE
clarify comments and defaults

### PR DESCRIPTION
**Pull request by ofek-ts**
Coming from a standpoint of someone who was completely new to this project, these are changes that would've made my life (and I hope other newcomings) a bit easier.

* changed DB_NAME to default as that is the default when creating a ClickHouse database
* fixed a typo (as discussed with vgorkavenko)

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -1,14 +1,14 @@
 # App
 HTTP_PORT=8080
-DB_PORT=8123
+DB_PORT=8443
```
Comment (vgorkavenko) [inline .env.example:3]: `8123` is the standard HTTP port for a [newly baked local Clickhouse instance](https://clickhouse.com/docs/en/interfaces/http#) (if you don't change server config), and `8443` is for HTTPS. So I propose to leave the current value

File: .env.example [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
 DB_HOST=http://clickhouse
 DB_USER=default
 DB_PASSWORD=
-DB_NAME=stats
+DB_NAME=default
```
Comment (vgorkavenko) [inline .env.example:7]:
```suggestion
DB_NAME=default
```

File: .env.example [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
 DB_HOST=http://clickhouse
 DB_USER=default
 DB_PASSWORD=
-DB_NAME=stats
+DB_NAME=default 
 # Logs
 # log format can be 'simple' or 'json'
 LOG_FORMAT=simple
-# select a slot as close as possible to the last finalized: find last finalized epoch, copy last slot from it
+# select a slot as close as possible to the last finalized: find last finalized epoch, copy last epoch from it
```
Comment (vgorkavenko) [inline .env.example:11]:
```suggestion
# select an epoch as close as possible to the last finalized.
# you can find it here: 
#  - https://beaconcha.in/epochs
#  - https://beaconscan.com/epochs
```

File: .env.example (hunk 1 above)
Comment (ofek-ts) [reply .env.example:3]: Wouldn't HTTPS be preferred by default over HTTP if possible? Perhaps I'm missing something

Comment (vgorkavenko) [reply .env.example:3]: it makes no sense for a local instance that runs as a `docker-compose` service (the standard and fastest approach for this project). If you have an external Clickhouse (on another host or in a different environment), of course, HTTPS would be the best option for you

Comment (ofek-ts) [reply .env.example:3]: Oh! I completely missed that part. I actually went ahead and create a ClickHouse instance, and didn't notice we are actually running one locally as part of the compose project. I agree.

**Review by vgorkavenko — DISMISSED** [review]
Thanks for your contribution!

**Review by vgorkavenko — APPROVED** [review]

**Pull request merged**